### ISSUE TITLE
Does not work with App Sandbox

### ISSUE THREAD

**Issue by elemans** [CleanCocoa/CrashReporter#2]
The crash files generated by macOS are stored outside the App Sandbox, and therefore not accessible when the app operates within an App Sandbox.

**Comment by DivineDominion** [comment]
Yeah :( I think the next best thing we have is to intercept the crash and collect and save our own reports: https://github.com/ChimeHQ/Impact
